src/components/IGVBrowser.js

```diff
@@ -19,6 +19,13 @@ import { RequesterPaysModal } from 'src/workspaces/common/requester-pays/Request
 import IGVSessionModal from './IGVSessionModal';
 import { updateUrlWithSession, useIGVSessions } from './useIGVSessions';
 
+function processUrl(url, isSignedUrl) {
+  if (url && isGoogleURL(url) && isGoogleStorageURL(url) && isSignedUrl) {
+    return translateGoogleCloudURL(url);
+  }
+  return url;
+}
+
```

Comment:
is the change here the addition of `isSignedUrl` in the if clause?

Reply:
Yes, that's the essential change in this PR -- beyond refactoring by extracting the logic into a function and the new test.

Changing from igv.js 2 to 3 requires Terra UI to reproduce some logic previously internal to IGV.  PR #5404 did that for DRS URI and inadvertently broke non-DRS in the process.  Adding this `isSignedUrl` condition restores non-DRS while also retaining that DRS support in IGV 3.